Pipfile.lock

```diff
@@ -225,39 +225,20 @@
         },
         "psycopg2": {
```

Comment:
Separate point, several projects has switched to `psycopg2-binary` which seems easier to install as it doesn't compile code, opinions? Not for the scope of this PR of course.

Reply:
more information about this can be found here: https://github.com/psycopg/psycopg2/issues/674

Reply:
I would recommend staying with the current dependency for this project.